### DISCUSSION TITLE
Getting Contact Forces During Interactions

### DISCUSSION THREAD

**Discussion by ege-dgny** [StanfordVL/BEHAVIOR-1K#1037] [Q&A] [answered]
I want to test my force-based MPC, how can I get force/torque values relative to the gripper(s) of the robot during interactions?

**Comment by hang-yin** [comment, marked as answer]
Hi @ege-dgny , not sure if you're still following this, but here's one way to do it:
```
finger_impulses = RigidContactAPI.get_impulses(robot_finger_prim_paths, prim_paths_to_check)
```
If we were to check for all impulses, we can do something like:
```
all_impulses = RigidContactAPI.get_all_impulses(scene_idx)
```

**Reply by ege-dgny** [reply]
Thank you @hang-yin I was away for sometime so apologies for my late reply. I will work on implementing this. If any more questions come up would you mind me asking?
